src/Snackbar/SnackbarContent.js

```diff
@@ -0,0 +1,98 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import transitions from 'material-ui/styles/transitions';
+import { darkWhite, pink, fullGrey } from 'material-ui/styles/colors';
+
+export const styleSheet = createStyleSheet('SnackbarContent', (theme) => {
+  const { typography } = theme;
+  return {
+    root: {
+      borderRadius: 2,
+      backgroundColor: fullGrey,
+      padding: '0px 24px ',
+      lineHeight: '48px',
+      overflow: 'visible',
+      width: '100%',
+    },
+    content: {
+      fontSize: typography.fontSize,
+      fontFamily: typography.fontSize,
+      color: darkWhite,
+      opacity: 1,
+    },
+    action: {
+      float: 'right',
```

Comment:
I would rather use the flexbox model. I don't think that we need any float. It's less performant and flexible.